README.md

```diff
@@ -72,7 +72,7 @@ source /dev/stdin <<<"$(linera net helper 2>/dev/null)"
 # owned by the default wallet. (The helper function `linera_spawn_and_read_wallet_variables`
 # is used to set the two environment variables LINERA_{WALLET,STORAGE}.)
 linera_spawn_and_read_wallet_variables \
-linera net up
+linera net up --table-name table_main
```

Comment:
Elsewhere, this is called a "namespace" not a table (--> `--namespace main`).

This being said, do we want this argument? Running several `linera net up` in parallel is not going to work anyway.

Reply:
The alternative is the following: Delete the corresponding namespace on the storage at the end of each test.

Reply:
Deleting makes sense to me. And/or the namespace could be randomized. The script `linera_spawn_and_read_wallet_variables` already reads the stdout of `linera net up` to set up the storage properly.

Reply:
Randomized namespace is indeed another approach. But that would require changing the "linera net up" functionality.

What is sure is that the persistence of the database means that something has to change in the `README.md`. Having separate table-name (pardon `namespace`) was an approach and maybe another one is needed.

Reply:
Ok, I reverted the changes to the README.md files and put back the creation of a child process for the `linera net up`.